src/graphics/vulkan/vulkan_context.cpp

```
@@ -128,7 +129,7 @@ namespace canyon::graphics::vulkan {
             }
 
             uint32_t glfwExtensionCount = 0;
```

Clang (14.0.6)

[error] variable 'glfwExtensionCount' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)